pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -130,23 +130,21 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *v1.Pod, attemp
 
 // generatePodSandboxLinuxConfig generates LinuxPodSandboxConfig from v1.Pod.
 func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod, cgroupParent string) *runtimeapi.LinuxPodSandboxConfig {
-	if pod.Spec.SecurityContext == nil && cgroupParent == "" {
-		return nil
+	lc := &runtimeapi.LinuxPodSandboxConfig{
```

Comment:
Just to point out that Instead of returning nil, this now will return an empty `LinuxPodSandboxConfig`.
But that won't affect anything.